src/Ydb.Protos/src/Auth/V1/AuthService.cs

```diff
@@ -0,0 +1,7 @@
+namespace Ydb.Auth.V1;
```

Comment:
src/Ydb.Protos/src/Auth/V1/AuthService.cs - is not gook looked path.

What about src/Ydb.Protos/Auth/V1/AuthService.cs ? (without additional src)